### DISCUSSION TITLE
All dokku configurations wiped out for some reason?

### DISCUSSION THREAD

**Discussion by josephchan91** [dokku/dokku#5116] [Q&A]
I recently added the public key of a developer on my team to my digital ocean ubuntu authorized_keys file and using 'dokku ssh-keys' also. He was able to deploy changes to the app. Today when I tried to fetch the latest commit from the dokku repo, it suddenly wasn't working. Tried removing and adding remote and nothing would happen on sourcetree. After ssh-ing into my server it appeared that my authorized_keys file was gone, all my app's config env vars were cleared, and all the domain/proxy/certs were gone too. I had to reconfigure and redeploy everything. I'd just like to know what I might have done to cause all of this to happen?

**Comment by josegonzalez** [comment]
Not knowing anything about your server setup, if you were using an older digitalocean image that still somehow had the web installer running, someone may have blanked your server by entering their own credentials. We've long since removed that web installer ui from Dokku because of the security risk (folks would forget to complete install through it).

If you didn't have that installed, then someone must have run commands to remove your sites. Do you have any extra monitoring on the server, or on the sites that are running? Maybe that can be used to pinpoint when this happened.

I would suggest shipping your server logs (/var/log/auth.log) offsite and monitoring those going forward.
